runtime/src/accounts_background_service.rs

```diff
@@ -55,7 +55,7 @@ pub struct SendDroppedBankCallback {
 
 impl DropCallback for SendDroppedBankCallback {
     fn callback(&self, bank: &Bank) {
-        if let Err(e) = self.sender.send((bank.slot(), bank.bank_id())) {
+        if let Err(e) = self.sender.try_send((bank.slot(), bank.bank_id())) {
```

Comment:
Seems scary. So if the send fails, what happens to the bank? Doesn't this mean the slot is never purged from AccountsDb?

Reply:
I see this error shows up when the validator is shutting down - the signal is disconnected. In such case, both `send` and `try_send` will fail. However, it shouldn't matter here. Right?
Another difference between `send` and `try_send` is when the channel is full. The question is that should we hang in accounts_background_service when the channel is full?

Reply:
https://github.com/solana-labs/solana/blob/c322842257fa7fcbea9daa04028dcad941d3aa95/ledger/src/bank_forks_utils.rs#L162
This channel is unbounded - full shouldn't happen? 
@carl What do you think about this change?

Reply:
Benchmarks shows that there is not significant performance difference between *send* and *try_send*. However, *try_send* seems to be 1ns faster. :)

```
test bench_bank_signal_send                              ... bench:          31 ns/iter (+/- 5)
test result: ok. 0 passed; 0 failed; 0 ignored; 1 measured; 6 filtered out; finished in 4.31s

test bench_bank_signal_try_send                          ... bench:          30 ns/iter (+/- 5)
test result: ok. 0 passed; 0 failed; 0 ignored; 1 measured; 6 filtered out; finished in 5.24s
```